CHANGELOG.md

```diff
@@ -109,6 +109,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Changed
 - Enable `./gradlew build` on MacOS by disabling bcw tests ([#7303](https://github.com/opensearch-project/OpenSearch/pull/7303))
 - Moved concurrent-search from sandbox plugin to server module behind feature flag ([#7203](https://github.com/opensearch-project/OpenSearch/pull/7203))
+- Block check type from METADATA_WRITE to METADATA_READ for index clear cache APIs ([#7303](https://github.com/opensearch-project/OpenSearch/pull/7303))
```

Comment:
Maybe reword this as something like "Allow access to file cache clear APIs for read only indexes" ?